src/Tribe/Plugin_Dependencies.php

```diff
@@ -0,0 +1,183 @@
+<?php defined( 'WPINC' ) or die;
+
+if (! class_exists( 'Tribe__Plugin_Dependencies' ) ) {
+	/**
+	 * Tracks which tribe plugins are currently activated
+	 */
+	class Tribe__Plugin_Dependencies {
+
+		/**
+		 * An multidimensional array of active tribe plugins in the following format
+		 *
+		 * array(
+		 *  'class'   => 'main class name',
+		 *  'version' => 'version num',
+		 *  'path'    => 'Path to the main plugin/bootstrap file'
+		 * )
+		 */
+		protected $active_plugins = array();
+
+		protected static $instance;
+
+		/**
+		 * Static Singleton Factory Method
+		 *
+		 * @return Tribe__Plugin_Dependencies
+		 */
+		public static function instance() {
+			if ( ! isset( self::$instance ) ) {
+				$className      = __CLASS__;
+				self::$instance = new $className;
+			}
+
+			return self::$instance;
+		}
+
+		public function __construct() {
+			$this->add_legacy_plugins();
+		}
+
+
+		/**
+		 * Registers older plugins that did not use this class
+		 */
+		private function add_legacy_plugins () {
```

Comment:
Space fix needed!

For functions, we don't add a space before the parenthesis. For control structures we do.